docs/source/advanced/transfer_learning.rst

```diff
@@ -119,3 +128,53 @@ Here's a model that uses `Huggingface transformers <https://github.com/huggingfa
             h_cls = h[:, 0]
             logits = self.W(h_cls)
             return logits, attn
+
+----------
+
+**********
+Finetuning
+**********
+
+For finetuning, Lightning provides a :class:`~pytorch_lightning.callbacks.finetuning.BackboneFinetuning` callback that can integrate the finetune strategy
```

Comment:
I don't think we should mention `BackboneFinetuning`. This isn't really based on a strong heuristic and it is quite opinionated. I added this at a time when users need an example but I know personally feel we should depreciate it.

Reply:
@rohitgr7 I would remove this one.